guides/17_Migration_to_3_0.md

```diff
@@ -53,4 +58,68 @@ handle them please remove.
 - AudioCallEnded
 - TURNMeetingEnded
 - StateMachineTransitionFailed
-- AudioDeviceSwitched
\ No newline at end of file
+- AudioDeviceSwitched
+
+### Updating `EventController` use cases
+We have de-coupled the `EventController` from `AudioVideoController`. Check below for the new addtions.
+```typescript
+interface EventController {
+  // Adds an observer for event published to this controller.
```

Comment:
Sorry, this feels a bit confusing to me. Should we simply say: "Adds an observer to receive events." ? Same for removeObserver.

Reply:
Yeah I had some weird wording here, updated